example/index.html

```diff
@@ -35,286 +35,760 @@ <h1 class="title">Gridly</h1>
                 </div>
             </section>
             <section class="demo">
-
                 <div class="row">
-                    <div class="col">One column</div>
+                    <div class="col">
+                        <div class="content">One column</div>
+                    </div>
                 </div>
                 <div class="row">
-                    <div class="col col-third">1 / 3</div>
-                    <div class="col">2 / 3</div>
+                    <div class="col col-half">
+                        <div class="content">column-half</div>
+                    </div>
+                    <div class="col col-quarter">
+                        <div class="content">column-quarter</div>
+                    </div>
+                    <div class="col col-quarter">
+                        <div class="content">column-quarter</div>
+                    </div>
                 </div>
                 <div class="row">
-                    <div class="col col-tenth">1 / 10</div>
-                    <div class="col col-fifth">1 / 5</div>
-                    <div class="col col-quarter">1 / 4</div>
-                    <div class="col">9 / 20</div>
+                    <div class="col col-third">
+                        <div class="content">1 / 3</div>
+                    </div>
+                    <div class="col">
+                        <div class="content">2 / 3</div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col col-tenth">
+                        <div class="content">1 / 10</div>
+                    </div>
+                    <div class="col col-fifth">
+                        <div class="content">1 / 5</div>
+                    </div>
+                    <div class="col col-quarter">
+                        <div class="content">1 / 4</div>
+                    </div>
+                    <div class="col">
+                        <div class="content">9 / 20</div>
+                    </div>
                 </div>
                 <div class="row">
                     <div class="col">Two</div>
```

Comment:
Is there a specific reason why you skipped this or you just missed it? :thought_balloon: